test/fixtures/expected.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+const CID = require('cids')
+
+module.exports = {
+  'array-link': new CID('zdpuAsZ8roJvCazJFXUnSULspbXfxrLkUkNxy6SToaiDpEiap'),
+  'empty-array': new CID('zdpuAtQy7GSHNcZxdBfmtowdL1d2WAFjJBwb6WAEfFJ6T4Gbi'),
+  'empty-obj': new CID('zdpuAyTBnYSugBZhqJuLsNpzjmAjSmxDqBbtAqXMtsvxiN2v3'),
+  foo: new CID('zdpuApbZ3Kmyxwa5rZ3mKHtQKkdHLKwHCXHTLbDSfEDfwkmen'),
```

Comment:
why did this one and fixtures/foo.cbor get changed?

Reply:
It's in the [commit message](https://github.com/ipld/js-ipld-dag-cbor/pull/104/commits/b8e587a8f506468755d9d8f08ca20a5998fbf78c): The CBOR binary file was modified
as the number `9` was encoded as `9.0` which isn't correct.